services/event_service.go

```diff
@@ -410,15 +411,16 @@ func (e *EventService) createEvent(ctx context.Context, endpoints []datastore.En
 	}
 
 	event := &datastore.Event{
-		UID:       ulid.Make().String(),
-		EventType: datastore.EventType(newMessage.EventType),
-		Data:      newMessage.Data,
-		Raw:       newMessage.Raw,
-		Headers:   e.getCustomHeaders(newMessage.CustomHeaders),
-		CreatedAt: time.Now(),
-		UpdatedAt: time.Now(),
-		Endpoints: endpointIDs,
-		ProjectID: g.UID,
+		UID:         ulid.Make().String(),
+		EventType:   datastore.EventType(newMessage.EventType),
+		Data:        newMessage.Data,
+		Raw:         newMessage.Raw,
+		Headers:     e.getCustomHeaders(newMessage.CustomHeaders),
+		QueryParams: newMessage.QueryParams,
```

Comment:
@surendratiwari3 I think It's better to limit this to the ingest path alone. I can't think of a good use-case for doing this via the API. Also, the contract will be confusing as to what is for our REST API and what should be proxied.